fabric-events-interaction-v0/src/main/resources/fabric.mod.json

```diff
@@ -25,7 +25,7 @@
       "net.fabricmc.fabric.impl.event.interaction.InteractionEventsRouter"
     ],
     "client": [
-      "net.fabricmc.fabric.impl.event.interaction.InteractionEventsRouterClient"
+      "net.fabricmc.fabric.test.client.event.interaction.InteractionEventsRouterClient"
```

Comment:
Surely it's not supposed to be in the test mod?

Reply:
Opps, lol.